extensions/markdown-language-features/package.json

```diff
@@ -99,7 +99,7 @@
       "editor/title": [
         {
           "command": "markdown.showPreviewToSide",
-          "when": "editorLangId == markdown && !notebookEditorFocused",
+          "when": "editorLangId == markdown && !notebookEditorFocused && !hasCustomMarkdownView",
```

Comment:
[nit] Can we call this `hasCustomMarkdownPreview` instead? That way it's super clear this is about the md preview

Otherwise looks good to me

Reply:
Updated, thanks!